Solutions/CohesitySecurity/readme.md

```diff
@@ -0,0 +1,20 @@
+# Cohesity SIEM/SOAR Integration with Sentinel
+
+This code integrates Cohesity Helios Cloud solution with MS Sentinel
+
+### Build Instructions
+1. Follow Tools/Create-Azure-Sentinel-Solution/V2/README.md:7 to setup build prerequisites
+2. Edit cohesity.config to replace these values with your owm
```

Comment:
@yinghuang123, please fix the typo "owm" to "own", thanks.

Reply:
The file was removed but the content was fixed